plugins/inputs/snmp/snmp_test.go

```diff
@@ -145,9 +145,9 @@ func TestFieldInit(t *testing.T) {
 		{".1.2.3", "foo", "", ".1.2.3", "foo", ""},
 		{".iso.2.3", "foo", "", ".1.2.3", "foo", ""},
 		{".1.0.0.0.1.1", "", "", ".1.0.0.0.1.1", "server", ""},
-		{"TEST::server", "", "", ".1.0.0.0.1.1", "server", ""},
-		{"TEST::server.0", "", "", ".1.0.0.0.1.1.0", "server.0", ""},
-		{"TEST::server", "foo", "", ".1.0.0.0.1.1", "foo", ""},
+		// {"TEST::server", "", "", ".1.0.0.0.1.1", "server", ""},
+		// {"TEST::server.0", "", "", ".1.0.0.0.1.1.0", "server.0", ""},
+		// {"TEST::server", "foo", "", ".1.0.0.0.1.1", "foo", ""},
```

Comment:
You know I don't like changing tests.. Why is this?

Reply:
Nevermind, just saw your PR comment.

Reply:
All tests are back to normal. Thank you for your patience :)